src/components/settings-dialog.tsx

```diff
@@ -12,11 +14,48 @@ export function SettingsDialog({ ...props }: React.ComponentProps<typeof Dialog>
           </DialogDescription>
         </DialogHeader>
 
-        <div className="grid grid-cols-2 gap-6 min-h-96">
-          <div>First column</div>
-          <div>Second column</div>
+        <div className="grid grid-cols-2 gap-6">
+          <div className="grid gap-3">
+          <span>Sensibilidade</span>
+          <Slider defaultValue={[1]} max={5} step={0.01} />
+          </div>
+          <div className="grid gap-3">
+          <span>Campo de visão</span>
+          <Slider defaultValue={[80]} max={110} step={1} />
+          </div>
+          <div className="grid gap-3">
+          <span>Tamanho do minimapa</span>
+          <Slider defaultValue={[50]} max={200} step={1} />
+          </div>
+          <div className="grid gap-3">
+          <span>Zoom do minimapa</span>
+          <Slider defaultValue={[1]} max={3} step={0.5} />
+          </div>
+        </div>
+
+        <span>Atalhos (clique para editar)</span>
+        <div className="grid grid-cols-2 gap-6">
+          <div className="flex flex-col gap-3 rounded-2xl border border-gray-850 px-4 py-3">
```

Comment:
Prefer to use the colors from the Shadcn palette as they handle the dark/light mode. Using the `border` already applies the `border-border` color.

```suggestion
          <div className="flex flex-col gap-3 rounded-2xl border px-4 py-3">
```